run_filetypeone.py

```
@@ -0,0 +1,50 @@
+﻿#!/usr/bin/env python3
```

Ruff (0.15.21)

[warning] Shebang should be at the beginning of the file

(EXE005)